### PR TITLE
[12.0][MIG] account_partner_required

### PR DESCRIPTION
**Pull request by rven**
This module depends on #702

### CODE REVIEW

**Review by tbaden — APPROVED** [review]

**Review by cfreyse — APPROVED** [review]

File: account_partner_required/readme/CONFIGURE.rst [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[ This file is optional, it should explain how to configure
```
Comment (pedrobaeza) [inline account_partner_required/readme/CONFIGURE.rst:1]: Please remove this text

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

File: account_partner_required/readme/CONFIGURE.rst (hunk 1 above)
Comment (tbaden) [reply account_partner_required/readme/CONFIGURE.rst:1]: @rven requested change to approve into your fork https://github.com/rven/account-financial-tools/pull/4

**Review by alexis-via — APPROVED** [review]
Tested, works fine.

File: account_partner_required/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Account partner required",
+    "version": "12.0.1.0.0",
+    "category": "Accounting",
+    "license": "AGPL-3",
+    "summary": "Adds an option 'partner policy' on account types",
+    "author": "ACSONE SA/NV,Akretion,Odoo Community Association (OCA)",
+    "website": "http://acsone.eu/",
```
Comment (nikul-serpentcs) [inline account_partner_required/__manifest__.py:15]: @rvalyi URL Must be `https://github.com/OCA/account-financial-tools`

**Review by nikul-serpentcs — CHANGES_REQUESTED** [review]
Netpicking

**Comment by alexis-via** [comment]
FYI, I took into account the remarks of @nikul-serpentcs and @pedrobaeza in a post-merge commit.

**Pull request merged**